dwave/system/composites/embedding.py

```diff
@@ -33,16 +33,19 @@
 
 
 class EmbeddingComposite(dimod.ComposedSampler):
-    """Composite that maps problems to a structured sampler.
+    """Maps problems to a structured sampler.
 
-    Enables quick incorporation of the D-Wave system as a sampler by handling
-    minor-embedding of the problem into the D-Wave system's :term:`Chimera`
-    graph. A new minor-embedding is calculated using the given `find_embedding`
-    function each time one of its sampling methods is called.
+    Automatically minor-embeds a problem into a structured sampler such as a
+    D-Wave system. A new minor-embedding is calculated each time one of its
+    sampling methods is called.
+
+    See also the :class:`.AutoEmbeddingComposite` class, which first tries to submit
```

Comment:
I would just move this to the bottom
```
Examples:
    ...

See Also:
    :class:`.AutoEmbeddingComposite` short description
    :class:`.FixedEmbeddingComposite` short description
    ...
```

Reply:
My reasoning here was that I think we want to encourage first-timers to move to AutoEmbeddingComposite rather than EmbeddingComposite. If we don't have such a preference, then it seems to me better to just spell out carefully what each class does, we don't usually refer from each class to other classes in the same family, users are expected to see what classes there are and choose the right one. Otherwise as we add classes, we need to update all the others for each new one

Reply:
I don't think we want users to move to AutoEmbeddingComposite. I understand it's use in a stack like dwave-hybrid, but if a user has a chimera-structured problem they should user the underlying structured sampler directly. If it is not, then the chances of it being "accidentally" chimera-structured are  extremely small.

Reply:
Okay, do you see a good case for maintaining links in each class for all the others? Otherwise I prefer to just accurately describe each one in its own class, like we do for most classes in families.

Reply:
I do like "See also" even without the descriptions. I find it very helpful when I am looking through a family of related functions/methods for the one that best fits my use case. That said there is obviously a judgement call here and I trust yours.

Reply:
I added a paragraph of general guidance on selecting an embedding composite that gives links as examples, not a comprehensive list. That should provide most the relevant coverage, will likely be copied in to any new classes and stay up-to-date without the risk that new classes won't be added to lists in previous classes.

Reply:
I think this is actually the worst of both worlds. Maybe just say nothing, we could move this paragraph to the module-level

Reply:
Happy to.